torchbenchmark/models/llama/__init__.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# This software may be used and distributed according to the terms of the GNU General Public License version 3.
+
+
+
+from ...util.model import BenchmarkModel
+from torchbenchmark.tasks import NLP
+import torch
+from .model import ModelArgs, Transformer
+import torch
+
+class Model(BenchmarkModel):
+    DEFAULT_EVAL_BSIZE = 32
+    task = NLP.LANGUAGE_MODELING
+
+    def __init__(self, test, device, jit=False, batch_size=DEFAULT_EVAL_BSIZE, extra_args=[]):
```

Comment:
Let's keep `batch_size=None`, in the parent class, it will setup the correct batch_size based on the value of `test`.

Reply:
Fixed

Reply:
If I set batch size to anything except something like 32 I get this error

```
Traceback (most recent call last):
  File "/home/ubuntu/benchmark/run.py", line 298, in <module>
    m = Model(device=args.device, test=args.test, jit=(args.mode == "jit"), batch_size=args.bs, extra_args=extra_args)
  File "/home/ubuntu/benchmark/torchbenchmark/util/model.py", line 20, in __call__
    obj = type.__call__(cls, *args, **kwargs)
  File "/home/ubuntu/benchmark/torchbenchmark/models/llama/__init__.py", line 17, in __init__
    super().__init__(test=test, device=device, jit=jit, batch_size=batch_size, extra_args=extra_args)
  File "/home/ubuntu/benchmark/torchbenchmark/util/model.py", line 84, in __init__
    self.determine_batch_size(batch_size)
  File "/home/ubuntu/benchmark/torchbenchmark/util/model.py", line 216, in determine_batch_size
    raise NotImplementedError(f"Test {self.test} is not implemented.")
NotImplementedError: Test eval is not implemented.
```